### PR TITLE
Update palma.ini

### PR DESCRIPTION
**Pull request by zuphilip**

### CODE REVIEW

File: examples/palma.ini [hunk 1]
```diff
@@ -26,7 +26,7 @@ id = ":1"
 ;~ ssh = "ssh palma@localhost";
 
 [general]
-; Enable or disable password authorization (default: true).
+; Enable or disable password authorization (default: false).
```
Comment (stweil) [inline examples/palma.ini]: If nothing is set here, the code in `DBConnector.class.php` will require password authorization, so this change does not look correct.

File: examples/palma.ini [hunk 2]
```diff
@@ -6,7 +6,7 @@
 ; Copy into the same directory as index.php.
 ;
 ; The entries here are available as PHP constants:
-,
+;
```
Comment (stweil) [inline examples/palma.ini]: Good catch.

File: examples/palma.ini (hunk 1 above)
Comment (zuphilip) [inline examples/palma.ini]: But default is doing nothing and then the line below will set it to `false`.

Comment (zuphilip) [inline examples/palma.ini]: Okay, I see this is just an example configuration.

**Comment by zuphilip** [comment]
Please have a look at the new version.

**Comment by stweil** [comment]
Thanks, I merged those fixes.

**Pull request merged**